src/content/community/releases/_index.en.md

```diff
@@ -4,6 +4,53 @@ title = "Releases"
 weight = 40
 +++
 
+## v0.12.0
+
+### New features
+
+* Added a new `subctl uninstall` command that removes all Submariner components and dataplane artifacts, such as iptables rules and routing
+table entries, from a cluster.
+* Added a new `subctl unexport` command that stops exporting a previously exported service.
+* Added new `subctl cloud prepare` and `subctl cloud cleanup` commands for the Red Hat OpenStack Platform (RHOS).
+* New Globalnet metrics were added:
+  * Count of global Egress IPs allocated at Cluster scope, namespace scope, and for selected pods per CIDR.
+  * Count of global Ingress IPs allocated for Pods/Services per CIDR.
+* Added support for Globalnet objects verification using subctl diagnose command.
+* Added support for `--broker-namespace` flag while deploying the Broker.
+* Added support for running `subctl diagnose` on single node clusters.
+* Added support for running `subctl diagnose` from a pod in cluster.
+* `subctl cloud prepare` now deploys a dedicated gateway node as a default option on GCP and OpenStack platforms.
+* `subctl show` now shows information about the Broker CR in the cluster.
+* `subctl gather` now collects Globalnet information.
+* `subctl diagnose` displays a warning when a generic CNI network plugin is detected.
+* New Service Discovery metric was added:
+  * Count of DNS queries handled by Lighthouse plugin.
+
+### Bug fixes
+
+* Calico is now correctly detected when used as a third-party plugin in OpenShift.
+* Services without selectors can now be resolved across the ClusterSet.
+* `subctl diagnose firewall inter-cluster` command will now work correctly for VXLAN cable driver.
```

Comment:
This is a pretty clear example of https://kubernetes.io/docs/contribute/style/style-guide/#starting-a-sentence-with-a-component-tool-or-component-name. I think we can do the fix recommended there:

```suggestion
* The `subctl diagnose firewall inter-cluster` command will now work correctly for VXLAN cable driver.
```

Or make the structure parallel to the similar lines above:

```suggestion
* `subctl diagnose firewall inter-cluster` now works correctly for VXLAN cable driver.
```